packages/api/endpoints/granules.js

```diff
@@ -678,6 +680,53 @@ const associateExecution = async (req, res) => {
   });
 };
 
+/**
+ * Update a batch of granule's collectionId to the new collectionId
+ * in PG and ES
+ *
+ * @param {Object} req - express request object
```

Comment:
We should enable typing/ts-chec (zod will help with typing the incoming event objects, in theory too)